src/app/check-recognizer-results/check-recognizer-results.page.html

```diff
@@ -0,0 +1,28 @@
+<ion-header>
+  <ion-toolbar>
+      <ion-buttons slot="start">
+          <ion-back-button defaultHref="home"></ion-back-button>
+      </ion-buttons>
+      <ion-title  *ngIf="checkResult.checkType" >
+        {{checkResult.checkType}} Check Recognizer Results 
+      </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content padding>
+  <div *ngIf="photoUri" style="margin: 1%; width: 98%; height: 50%; text-align: center; background-color: rgb(245, 245, 245)">
+    <img style="max-width:100%; max-height:100%; vertical-align: middle;" [src]="photoUri"/>
+  </div>
+
+  <li *ngFor="let item of displayFields | keyvalue" style="list-style: none;">
+      <ion-item style="display: block;">
+          <ion-label style="margin-left: 0.5em; font-size: 0.8em; font-weight: 600;">{{item.key}}</ion-label>
+          <ion-label style="margin-left: 0.5em; font-size: 0.8em; font-weight: 400; white-space: pre-wrap">{{item.value}}</ion-label>
+      </ion-item>
+  </li>
+
+</ion-content>
+
+<ion-footer>
+
+</ion-footer>
```

Comment:
Next file https://github.com/doo/scanbot-sdk-example-ionic/blob/ec9f0cc96af1dc5edf61e8a7c44f2d2808a5c519/src/app/check-recognizer-results/check-recognizer-results.page.scss is Empty Can you check?

Reply:
It's the css file that belongs to the HTML page. I wanted to keep the structure but yeah we can get rid of it